packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -347,160 +357,170 @@ export default class OperatorModeStackItem extends Component<Signature> {
         class={{cn 'card' edit=(eq @item.format 'edit')}}
         {{ContentElement onSetup=this.setupContainerEl}}
       >
-        <Header
-          @title={{this.headerTitle}}
-          class={{cn 'header' header--icon-hovered=this.isHoverOnRealmIcon}}
-          {{on
-            'click'
-            (optional (if this.isBuried (fn @dismissStackedCardsAbove @index)))
-          }}
-          data-test-stack-card-header
-        >
-          <:icon>
-            {{#if this.headerIcon.URL}}
-              <RealmIcon
-                @realmIconURL={{this.headerIcon.URL}}
-                @realmName={{this.realmName}}
-                class='header-icon'
-                data-test-boxel-header-icon={{this.headerIcon.URL}}
-                {{on 'mouseenter' this.headerIcon.onMouseEnter}}
-                {{on 'mouseleave' this.headerIcon.onMouseLeave}}
-              />
-            {{/if}}
-          </:icon>
-          <:actions>
-            {{#if (eq @item.format 'isolated')}}
+        {{#if this.loadCard.isRunning}}
```

Comment:
this diff didn't render so nicely, but basically just wrapping the template with a guard to ensure that the card resource finishes loading before we render this.